### DISCUSSION TITLE
Inquiry for using the PDFScraper

### DISCUSSION THREAD

**Discussion by summywong-developer** [erikkastelec/PDFScraper#7] [Q&A] [answered]
Hi, erikkastelec, 
Could you please teaching me how to use PDFScraper on Mac? 

Since I am currently involved in technical research and in need of using this software for doing the steps of data extraction. Please let me know if we have a time to have a call via Skype, Slack, or Disorder either.

**Comment by erikkastelec** [comment]
Hi, 

I am not available for one on one consultations about the project, but I will be more than happy to answer any specific questions you have about it.
If you tell me more about what kind of data you want to extract and in what format it will be used after extraction, I can point you in the right direction.

P.S. Sorry for the late reply

**Comment by summywong-developer** [comment]
Hi,

Thanks for your reply.

Since I want to crawl the data from via website and pdf file and export it into excel format to analyse.

I have tried octoparser, web scraper from google and google vision API and also python.

Do you have any suggestions? Like python and R library which are able to run on mac and window?

P.S: It's OKAY. I knew not everyone keep tracking on GitHub.




Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Reply by erikkastelec** [reply, marked as answer]
For extraction of text data  from the PDF  I used [pdfminer.six](https://github.com/pdfminer/pdfminer.six). If you need to extract data from tables than [camelot](https://github.com/camelot-dev/camelot) would be a better choice.

Both of the libraries are well documented (I only have Slovene documentation for mine), but you can still  take a look at how I used them in my library.

If you have PDF documents, which are in "image" form (you can't copy and paste from them) than I suggest you use [Tesseractt](https://github.com/tesseract-ocr/tesserac) to convert them into editable pdf format.
